addons/website/views/snippets/snippets.xml

```diff
@@ -539,6 +539,10 @@
             <we-button data-customize-website="">Full</we-button>
             <we-button data-customize-website="website.option_layout_boxed_variables" id="option_layout_boxed">Boxed</we-button>
         </we-select>
+        <we-row string="Code Injection" title="Enter code that will be added into every page of your site">
+            <we-button data-no-preview="true" data-open-custom-code-dialog="head">Head</we-button>
+            <we-button data-no-preview="true" data-open-custom-code-dialog="footer">Footer</we-button>
```

Comment:
I wonder if the label of those buttons should instead be `<head>` and `</body>` instead of `Head` and `Footer`. To check with @sbuhl